.github/workflows/ci.yaml

```diff
@@ -18,6 +18,10 @@ name: CI
       - '*'
     paths-ignore:
       - '**.md'
+    types:
+      - opened
+      - reopened
+      - synchronize
```

Comment:
According to https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#pull_request, this is the default, so we do not need to state this explicitly.